modules/EnsEMBL/Web/Component/Gene/FamilyProteins.pm

```diff
@@ -185,19 +185,20 @@ sub content_ensembl {
       { key => 'peptides', title => 'Proteins', width => '80%', align => 'left' },
     );
 
+    my $lookup = $prodname_to_url_lookup;
```

Comment:
`$prodname_to_url_lookup` is not initialised. 
Use `$hub->species_defs->production_name_lookup;` instead?

Reply:
Ah, good catch! Should be "prodnames_to_urls_lookup" (I renamed a couple of methods to make the direction of the mapping clearer).

Reply:
I don't think it really matters, since we don't calculate or display gene families any more, but i'll fix it for consistency's sake.